### PR TITLE
feat: Add rpc priority as an option

### PR DESCRIPTION
**Pull request by zoercai**

### CODE REVIEW

File: sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/cdc/mapper/ChangeStreamRecordMapperTest.java [hunk 1]
```diff
@@ -108,7 +108,7 @@ public void testMappingInsertStructRowToDataChangeRecord() {
         DataChangeRecord.newBuilder()
             .withPartitionToken("partitionToken")
             .withCommitTimestamp(Timestamp.ofTimeSecondsAndNanos(10L, 20))
-            .withTransactionId("transactionId")
+            .withServerTransactionId("transactionId")
```
Comment (thiagotnunes) [inline sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/cdc/mapper/ChangeStreamRecordMapperTest.java:111]: Is this something we missed?

File: sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/cdc/mapper/ChangeStreamRecordMapperTest.java [hunk 2]
```diff
@@ -138,7 +138,7 @@ public void testMappingDeleteStructRowToDataChangeRecord() {
...
```
Comment (thiagotnunes) [inline sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/cdc/mapper/ChangeStreamRecordMapperTest.java:141]: Is this something we missed from in @hengfengli 's PR?

**Review by thiagotnunes — APPROVED** [review]
Thanks for building this!

File: sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/cdc/mapper/ChangeStreamRecordMapperTest.java (hunk 1 above)
Comment (zoercai) [reply sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/cdc/mapper/ChangeStreamRecordMapperTest.java:111]: yeah currently it's not building

File: sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/cdc/mapper/ChangeStreamRecordMapperTest.java (hunk 2 above)
Comment (zoercai) [reply sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/cdc/mapper/ChangeStreamRecordMapperTest.java:141]: I think so, currently it's not building on my local without this

**Pull request merged**